packages/andromeda-non-fungible-tokens/src/crowdfund.rs

```diff
@@ -212,3 +207,53 @@ pub struct TierMetaData {
     /// Any custom extension used by this contract
     pub extension: TokenExtension,
 }
+
+#[andr_query]
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum QueryMsg {
+    #[returns(CampaignSummaryResponse)]
+    CampaignSummary {},
+    #[returns(TierOrdersResponse)]
+    TierOrders {
+        orderer: String,
+        start_after: Option<u64>,
```

Comment:
Mind adding a comment about what this is indexing (what the u64 value references)?

Reply:
Added